core/src/main/java/bisq/core/trade/statistics/TradeStatistics3.java

```diff
@@ -437,7 +437,12 @@ public boolean isValid() {
         }
         long maxTradeLimit = Coin.COIN.multiply(2).value;
         try {
-            maxTradeLimit = PaymentMethod.getPaymentMethodById(getPaymentMethod()).getMaxTradeLimitAsCoin(currency).value;
+            // We cover only active payment methods. Retired ones will not be found by getActivePaymentMethodById.
+            String paymentMethod = getPaymentMethod();
```

Comment:
`utACK` with a comment about line 441's var name not being `paymentMethodId`.  

It's clear from line 401's method declaration `public String getPaymentMethod()` that this is an ID (name).  But maybe line 401 could also be changed to `public String getPaymentMethodId()`?

Reply:
Actually its not the ID (ending with _ID) but the name of the payment method. It was used so far only for display. Would have been better to use the ID...

Reply:
Renamed it...